app/views/partials/channels-tab-basic-info.html

```diff
@@ -22,6 +22,40 @@
   </div>
 </div>
 
+<div class="form-group-2columns" ng-show="channel.type == 'http'">
+    <div class="form-group" style="float: left;" ng-class="{ 'has-error' : ngError.methods }">
+    <label><span>*</span>Allowed HTTP methods:</label>
+    <div class="checkbox">
+        <label><input type="checkbox" ng-model="methods.POST" ng-checked="methods.POST">POST</label>
```

Comment:
`ng-model` and `ng-checked` should not be used together:

> Note that this directive should not be used together with ngModel, as this can lead to unexpected behavior. - https://docs.angularjs.org/api/ng/directive/ngChecked

It should be one or the other, and by looking at the docs it seems like `ng-model` is more correct in this case.